src/test/java/com/exasol/adapter/document/DynamodbVsExasolTestDatabaseBuilder.java

```diff
@@ -0,0 +1,66 @@
+package com.exasol.adapter.document;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.sql.SQLException;
+import java.util.concurrent.TimeoutException;
+
+import com.exasol.bucketfs.BucketAccessException;
+
+/**
+ * DynamoDB specific test database builder.
+ */
+public class DynamodbVsExasolTestDatabaseBuilder extends ExasolTestDatabaseBuilder {
+    private static final String VIRTUAL_SCHEMAS_JAR_NAME_AND_VERSION = "document-virtual-schema-dist-0.3.0-dynamodb-0.5.0.jar";
+    private static final Path PATH_TO_VIRTUAL_SCHEMAS_JAR = Path.of("target", VIRTUAL_SCHEMAS_JAR_NAME_AND_VERSION);
+
+    public DynamodbVsExasolTestDatabaseBuilder(final ExasolTestInterface testInterface)
+            throws SQLException, IOException {
+        super(testInterface);
+    }
+
+    /**
+     * Uploads the dynamodb adapter jar to the exasol test container.
+     */
+    public void uploadDynamodbAdapterJar() throws InterruptedException, BucketAccessException, TimeoutException {
+        this.testInterface.uploadFileToBucketfs(PATH_TO_VIRTUAL_SCHEMAS_JAR, VIRTUAL_SCHEMAS_JAR_NAME_AND_VERSION);
+        uploadJacocoAgent();
+    }
+
+    public void createUdf() throws SQLException {// TODO make DynamoDB independent
+        final StringBuilder statementBuilder = new StringBuilder(
+                "CREATE OR REPLACE JAVA SET SCRIPT " + ADAPTER_SCHEMA + "." + UdfRequestDispatcher.UDF_PREFIX
+                        + "DYNAMO_DB" + "(" + AbstractDataLoaderUdf.PARAMETER_DOCUMENT_FETCHER + " VARCHAR(2000000), "
```

Comment:
It will be more compact if you add a static import for the constants. But now it's human-readable :+1: